src/CheckBox/README.md

```diff
@@ -0,0 +1,17 @@
+# Checkbox Component
+
+Checkbox based off of the [WAI-ARIA authoring guidelines for the checkbox component](https://www.w3.org/TR/wai-aria-1.1/#checkbox). The checkbox is an HTMl input element with added attributes to ensure proper functionality and accessibility. The checkbox is wrapped in an `asInput` component. The `asInput` wrapper is passed the input element as well as a second parameter set to `false` to ensure that the label is placed to the right of the checkbox.
+
+The following inputs should be passed into every checkbox component:
+* `name`: `name` attribute - type `String`
```

Comment:
:+1: for declaring the type beside each of these props, although I'd place it at the beginning of the line, e.g.:
```
`name` (string): `name` attribute
```